controllers/save_music.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../modules/file_handle'
+require_relative '../models/music_album'
+
+class MusicController
+  include HandleFiles
+
+  def list
+    HandleFiles.read('musics.json').map do |music|
+      obj = MusicAlbum.new(music['album'], music['genres'], music['genre_name'])
+      obj
```

Comment:
- Data not loading on the `app` initial load, it's only showing `List @music_album`. Please implement this functionality to view the saved music's data from `musics.json`. 🔢 
- For this, you need to implement the functionality the same as it's working for for `movies` module. 💯